hummingbot/strategy/pure_market_making/pure_market_making_v2.pyx

```diff
@@ -639,39 +649,33 @@ cdef class PureMarketMakingStrategyV2(StrategyBase):
             MarketBase maker_market = market_info.market
             LimitOrder limit_order_record
 
-        if isinstance(self.sizing_delegate, self.SINGLE_ORDER_SIZING_DELEGATES):
-            # Set the replenish time as current_timestamp + order replenish time
-            replenish_time_stamp = self._current_timestamp + self._filled_order_replenish_wait_time
-
-            active_orders = self.market_info_to_active_orders.get(market_info, [])
-            active_buy_orders = [x.client_order_id for x in active_orders if x.is_buy]
-            active_sell_orders = [x.client_order_id for x in active_orders if not x.is_buy]
+        # Set the replenish time as current_timestamp + order replenish time
+        replenish_time_stamp = self._current_timestamp + self._filled_order_replenish_wait_time
 
-            if self._enable_order_filled_stop_cancellation:
-                # If the filled order is a hanging order (not an active buy order)
-                # do nothing
-                if order_id not in active_buy_orders:
-                    return
+        active_orders = self.market_info_to_active_orders.get(market_info, [])
+        active_buy_orders = [x.client_order_id for x in active_orders if x.is_buy]
+        active_sell_orders = [x.client_order_id for x in active_orders if not x.is_buy]
 
-            # if filled order is buy, adjust the cancel time for sell order
-            # For syncing buy and sell orders during order completed events
-            for other_order_id in active_sell_orders:
-                if other_order_id in self._time_to_cancel:
+        if self._enable_order_filled_stop_cancellation:
+            # If the filled order is a hanging order, do nothing
```

Comment:
not sure what this means? can you clarify this logic?

Reply:
It's a bit hard to read the code comparison here as GitHub outlines it a bit. confusingly. Essentially, it is the logic here is same as existing, if the order that is filled is one of the hanging orders, we don't need to do anything here.

Reply:
It's a bit hard to read the code comparison here as GitHub outlines it a bit. confusingly. Essentially, it is the logic here is same as existing, if the order that is filled is one of the hanging orders, we don't need to do anything here.